rskj-core/src/main/java/co/rsk/net/discovery/PeerExplorer.java

```diff
@@ -228,6 +255,17 @@ private void handleFindNode(FindNodePeerMessage message) {
     private void handleNeighborsMessage(InetSocketAddress neighborsResponseAddress, NeighborsPeerMessage message) {
         Node connectedNode = this.establishedConnections.get(message.getNodeId());
 
+        logger.debug("handleNeighborsMessage - Handling neighbors message with " +
+                        "neighborsResponseAddress hostName: {}," +
+                        "neighborsResponseAddress port: [{}]," +
+                        "nodeId: [{}]," +
+                        "messageId: [{}], " +
+                        "nodesCount: [{}], " +
+                        "nodes: [{}], " +
+                        "connectedNode: [{}]"
+                , neighborsResponseAddress.getHostName(), neighborsResponseAddress.getPort(), message.getNodeId(),
+                message.getMessageId(), message.countNodes(), message.getNodes(), connectedNode);
+
         if (connectedNode != null) {
             logger.debug("Neighbors received from [{}]", connectedNode.getHexId());
```

Comment:
same here - we can extend the one - `logger.debug("Neighbors received from [{}] ...` and remove the one above and print a WARN if `connectedNode` is null

Reply:
Done